cmd/catalyst-node/catalyst-node.go

```diff
@@ -462,8 +494,8 @@ func protocol(r *http.Request) string {
 	return "http"
 }
 
-func queryMistForClosestNode(playbackID, lat, lon string) (string, error) {
-	url := fmt.Sprintf("http://localhost:%s/%s", mistUtilLoadPort, playbackID)
+func queryMistForClosestNode(playbackID, lat, lon, prefix string) (string, error) {
+	url := fmt.Sprintf("http://localhost:%d/%s+%s", mistUtilLoadPort, prefix, playbackID)
```

Comment:
Hmmmmmmmmmmm I'm trying to decide if we want to support non-prefixed playback. I think it's fine that we don't in this PR but I can imagine someone wanting to do something with that in the future

Reply:
done!

![i][i]

[i]: https://media2.giphy.com/media/AvMJCeu1EMmhG/giphy.gif